linux-tools/test_time/test_time.py

```diff
@@ -0,0 +1,196 @@
+#!/bin/python
+
+from autotest.client import test, utils
+from distutils.spawn import find_executable
+from autotest.client.shared import error
+
+
+class test_time(test.test):
+    """
+    Autotest module for time utility.
+    time - time is a simple command,it gives resource usage.
+    The time utility runs the specified program/command with the given arguments.
+    When program/command finishes,time writes a message to standard error giving
+    timing statistics about the program/command run.
+    These statistics consist of
+        (i)   The elapsed real time between invocation and termination,
+        (ii)  The user CPU time , and
+        (iii) The system CPU time .
+
+    @author:Basheer Khadarsabgari <basheer@linux.vnet.ibm.com>
+    """
+    version = 1
+    nfail = 0
+    seconds = 2
+    time_exe = ''
+
+    def install_check(self):
+        """
+        Install check for time
+        """
+        self.time_exe = find_executable('time')
+        if self.time_exe is None:
+            raise error.TestError('\n time utility not found')
+
+    def test_portability_option(self):
+        """
+        This method will verify the time utility output Format
+        Ex: "/usr/bin/time -p sleep 2"
+        """
+        cmd = "%s -p sleep %s" % (self.time_exe, self.seconds)
+        time_output = utils.commands.getoutput(cmd)
+        if utils.re.search(
```

Comment:
Instead of utils.commands, import commands and use commands.getoutput
Same for utils.re .